docs/windows.md

```diff
@@ -101,6 +101,12 @@ Artifacts work mostly the same way as on Linux. All paths get automatically mapp
         args: ["dir C:\\message"]   # List the C:\message directory
 ```
 
+Remember that [volume mounts on Windows can only target a directory](https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#storage) in the container, and not an individual file.
+
+## Limitations
+
+- Sharing process namespaces [doesn't work on Windows](https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#v1-pod) so you can't use the Process Namespace Sharing (pns) workflow executor.
```

Comment:
Can we add a similar comment on `docs/workflow-executors.md`?

Reply:
added